src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.spec.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import { isEqual } from 'lodash';
+
+import { DeploymentsDonutChartComponent } from './deployments-donut-chart.component';
+
+describe('DeploymentsDonutChartComponent', () => {
+  let component: DeploymentsDonutChartComponent;
+  let fixture: ComponentFixture<DeploymentsDonutChartComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [DeploymentsDonutChartComponent]
+    });
+
+    fixture = TestBed.createComponent(DeploymentsDonutChartComponent);
+    component = fixture.componentInstance;
+
+    component.pods = [{
+      status: {
+        phase: 'Running'
+      }
+    }, {
+      status: {
+        phase: 'Terminating'
+      }
+    }];
```

Comment:
Indentation here looks funny to me - should the inner object literals be indented one level more?